src/main/resources/application.yml

```diff
@@ -10,9 +10,9 @@ application:
   defaultAction:
     service: default-action
     body:
-      some_val: "Hello World"
+      someVal: "Hello World"
     query:
-      another_val: 123
+      anotherVal: 123
```

Comment:
Just to clarify - if these are missing, the service is still executed, right?

Reply:
Yes, body and query are optional